tests/tests_unit/rules/test_models/test_dms_architect_rules.py

```diff
@@ -403,6 +403,220 @@ def rules_schema_tests_cases() -> Iterable[ParameterSet]:
         id="View not in model",
     )
 
+    dms_rules = DMSRules(
+        metadata=DMSMetadata(
+            # This is a complete schema, but we do not want to trigger the full validation
+            schema_="partial",
+            space="my_space",
+            external_id="my_data_model",
+            version="1",
+            creator=["Anders"],
+            created="2024-03-17T11:00:00",
+            updated="2024-03-17T11:00:00",
+            default_view_version="1",
+        ),
+        properties=SheetList[DMSProperty](
+            data=[
+                DMSProperty(
+                    class_="Asset",
+                    property_="name",
+                    value_type="text",
+                    container="Asset",
+                    container_property="name",
+                    view="Asset",
+                    view_property="name",
+                ),
+                DMSProperty(
+                    class_="Asset",
+                    property_="timeseries",
+                    value_type="Timeseries:asset",
```

Comment:
Here we have View `Asset` that has property `timeseries` which points to ? `Timeseries` (simple type), but you want to add reverse property `asset` to `Timeseries` as object to write back? 

My worry here is that it breaks the pattern `space:View(version)` when referring to views. suddenly we have BaseType:reverseProperty as pattern, so my proposal would be to have following:

`BaseType(reverseProperty=...)` and `space:View(version=..., reverseProperty=...)

Reply:
Lets not include `reverse` here as that is already part of the relation. Anyways I am all ears for good suggestions, I think this is maybe easier to reuse `space:View(version=..., property=...)`.